source/docs/a1.why_rust.md

```diff
@@ -2,17 +2,17 @@ title: Why Rust?
 ---
 
 ## History of Rust
-Rust initially designed and developed by Mozilla employee **Graydon Hoare** as a personal project. Mozilla began sponsoring the project in 2009 and announced it in 2010. But the first stable release, Rust 1.0 released on May 15, 2015.
+Rust was initially designed and developed by Mozilla employee **Graydon Hoare** as a personal project. Mozilla began sponsoring the project in 2009 and announced it in 2010. But the first stable release, Rust 1.0 was released on May 15, 2015.
 
 ## Initial Goals
-The goal of Rust is to be a good language for creating highly concurrent and highly safe systems. Also, Rust is designed to provide speed and safety at the same time.
+The goal of Rust is to be a good programming language for creating highly concurrent, safe and performant systems.
 
 > "Rust is a systems programming language focused on three goals: safety, speed, and concurrency."  
 > \_\_ Rust Documentation
 
-Rust is very young and very modern language. It's a **compiled programming language** and it uses [LLVM](https://en.wikipedia.org/wiki/LLVM) on its backend. Also Rust is a **multi-paradigm programming language**, it supports imperative procedural, concurrent actor, object-oriented and pure functional styles. It also supports generic programming and meta programming, in both static and dynamic styles.
+Rust is very young and very modern language. It is a **compiled programming language** and it uses [LLVM](https://en.wikipedia.org/wiki/LLVM) on the backend. It's a **multi-paradigm programming language**, that is; it supports imperative procedural, concurrent actor, object-oriented and pure functional styles. It also supports generic programming and meta programming, in both static and dynamic styles.
```

Comment:
"It's a multi-paradigm programming language, that is; it supports..." not correct, right?